backend/inspirehep/alembic/8ba47044154a_add_indexed_to_records_citations_on_is_.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""Add indexed to records_citations on is_self_citation column"""
+
+from alembic import op
+
+revision = "8ba47044154a"
+down_revision = "5a0e2405b624"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
+    """Upgrade database."""
+    op.create_index(
+        "ix_records_citations_cited_id_citation_type",
+        "records_citations",
+        ["cited_id", "is_self_citation"],
+        unique=False,
+    )
+    op.create_index(
+        "ix_records_citations_citer_id_citation_type",
+        "records_citations",
+        ["citer_id", "is_self_citation"],
+        unique=False,
+    )
```

Comment:
don't you want to extend the existing indexes to add `is_self_citation` instead?

Reply:
Currently we have two indexes, with citer_id, cited_id and opposite one. They are quite big(2G). With our infrastructure additional small indexes would be more efficient than extending existing ones.